### PR TITLE
Create a-mir-formality.toml

### PR DESCRIPTION
**Pull request by nikomatsakis**
Is this all I need? Will this enable bors?

### CODE REVIEW

**Review by rylev — APPROVED** [review]
Looks good. 

This isn't everything needed for bors: https://forge.rust-lang.org/infra/docs/bors.html#adding-a-new-repository-to-bors

File: repos/rust-lang/a-mir-formality.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+bots = ["rustbot", "rfcbot", "bors"]
+
+[access.teams]
+lang = "maintain"
+lang-ops = "maintain"
+lang-docs = "maintain"
+types = "maintain"
+release = "maintain"
+infra = "maintain"
+infra-bors = "write"
```
Comment (rylev) [inline repos/rust-lang/a-mir-formality.toml:13]: This is probably unnecessary. If you added this just to get bors to work you can remove it.

File: repos/rust-lang/a-mir-formality.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description = "a model of MIR and the Rust type/trait system"
+bots = ["rustbot", "rfcbot", "bors"]
+
+[access.teams]
+lang = "maintain"
+lang-ops = "maintain"
+lang-docs = "maintain"
+types = "maintain"
+release = "maintain"
+infra = "maintain"
```
Comment (rylev) [inline repos/rust-lang/a-mir-formality.toml:12]: Is there a reason for these two to have maintenance permissions?

File: repos/rust-lang/a-mir-formality.toml (hunk 1 above)
Comment (Mark-Simulacrum) [inline repos/rust-lang/a-mir-formality.toml:13]:
```suggestion
```

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**